extras/parse_travis_log.py

```diff
@@ -899,7 +900,6 @@ def build_return_code(status_vera_init,
        (status_make_install) and \
        (status_tests) and \
        (number_of_msgs_in_summary(summary_vera) == 0) and \
-       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
This should be something like
```python
( not perform_cppcheck or number_of_msgs_in_summary(summary_cppcheck) == 0 )
```
Also, PEP8 prefers parentheses to continuation lines.